### DISCUSSION TITLE
Is there a possibility to remove the "Help menu"?

### DISCUSSION THREAD

**Discussion by Lazouoi** [TerriaJS/terriajs#7179] [Q&A] [answered]
After spending a lot of time I tried to remove the  Help menu using different tricks but none worked well. For example: 
1. Disabling the help button in the menu bar (didn't work)
2. Commenting last time of "HelpPanel". (Didn't work and the build failed).
3. Commenting in the StandUserUi (same results to 2)

Is it really possible to remove this help menu? Why is it deeply integrated? Or I'm missing.

**Comment by na9da** [comment, marked as answer]
Answered here - https://github.com/TerriaJS/terriajs/issues/7180
